SUMMARY.md

```diff
@@ -23,6 +23,7 @@
   * [Registers](operations/registers.md)
   * [Tables](operations/tables.md)
   * [Webhook endpoints](operations/webhookendpoints.md)
+  * [Areas](operations/areas.md)
```

Comment:
@MirkoC Can you move this to the top please, they should be in alphabetical order.